lib/matplotlib/axes/_axes.py

```diff
@@ -5326,12 +5314,26 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             The Colormap instance or registered colormap name used to map
             scalar data to colors. This parameter is ignored for RGB(A) data.
 
-        norm : `~matplotlib.colors.Normalize`, optional
+        norm : str or `~matplotlib.colors.Normalize`, optional
             The `.Normalize` instance used to scale scalar data to the [0, 1]
-            range before mapping to colors using *cmap*. By default, a linear
+            range before mapping to colors using *cmap*.  By default, a linear
             scaling mapping the lowest value to 0 and the highest to 1 is used.
             This parameter is ignored for RGB(A) data.
 
+            The normalization can also be given as a str, which should be a
+            scale name.  In that case, a normalization class is dynamically
```

Comment:
No end user knows what scale names are supported. We should either link that to some place or hard-code the list of current scales here.

Reply:
I hardcoded the list, as is current done in the doc for set_xscale.